File: challenge-3.py
def chall_three (array) :
    count = 0
    for i in array:
        # print(type(i))
        if type(i) == str:
            count += len(i)
        else:
            count += len(str(i))
        #     count += len(i)
        # else:
        #     count += len(i)
    
    print(count)
    
    
arr = [453, "hello", 293, True]
chall_three(arr)

File: coding-challenge/challenge-3.js
arr = [453, "hello", 293, true]

function arrayCounter(array) {
    let sepArray = array.map((value) => {
        let stringVal = value.toString()
        let splitVal = stringVal.split("")
        let nums = splitVal.push()
        return nums
    })
    let sum = sepArray.reduce(function (accumulator, currentValue){
        return accumulator + currentValue
    })
    console.log(sum)
}

arrayCounter(arr)